lungmask/utils.py

```diff
@@ -98,7 +98,7 @@ def __len__(self):
         return len(self.dataset)
 
     def __getitem__(self, idx):
-        return self.dataset[idx, None, :, :].astype(np.float)
+        return self.dataset[idx, None, :, :].astype(float)
```

Comment:
could you please change this to `np.float32`? just because it's a bit more expressive...

Reply:
hi @JoHof - I'm very happy to change it. I think this will change the behaviour from what it was before because I think np.float was an alias to float which in python 64 bit. If we want to keep the same behaviour as previously whilst still being expressive then we can have np.float64. It's totally up to you Johannes.

Reply:
On the torch side the input will be casted to float32 anyway: https://github.com/JoHof/lungmask/blob/3e4417b02080bec9d2e1e95d5a683b6976898292/lungmask/mask.py#L68
so there won't be a change in behaviour. Using float32 here saves memory and one type cast. So yes, please change it to `np.float32`. Your help is highly appreciated!

Reply:
Ahh, perfect. I'll make the change now.

Reply:
Hi @JoHof - it should be ready to merge now